### ISSUE TITLE
[connectivity_plus] doesn't handle internet status on Chrome

### ISSUE THREAD

**Issue by Graaggas** [fluttercommunity/plus_plugins#856]
## System info

Platform the Issue occurs on: chrome 101.0.4951.54
Plugin name: connectivity_plus
Plugin version: 2.3.0

## Steps to Reproduce

Good day!

I'm working on a web app and using connectivity_plus to test my internet connection. It works properly for Safari and Firefox, but for Chrome, switching off the WiFi connectivity doesn't change the ConnectivityResult. It's permanently 'mobile'. As we can see in the logs, if we switch off the connection, the connectivity result is the same. Is there any solution for Chrome?

<details>
<summary>Logs</summary>

```
 ❍ [Model] checking connection...  
[   +1 ms]  ❍ [Connection scope] checking connection...  
[        ]  ❍ [UseCase] checking connection...  
[        ]  ❍ [Widget] isOnline: null
[   +2 ms]  ❍ [Monitor] connection state:  mobile
[        ] ==> connectionState = ConnectivityResult.mobile
[        ]  ❍ [Monitor] comparing connection state...  
[        ]              connection: true 
[        ]              connectivity result: ConnectivityResult.mobile
[        ] ==> value = ConnectivityResult.mobile
[+21908 ms] ==> connectionState = ConnectivityResult.mobile
[        ]  ❍ [Monitor] comparing connection state...  
[        ]              connection: true 
[        ]              connectivity result: ConnectivityResult.mobile
[        ] ==> value = ConnectivityResult.mobile
[        ] ==> connectionState = ConnectivityResult.mobile
[        ]  ❍ [Monitor] comparing connection state...  
[        ]              connection: true 
[        ]              connectivity result: ConnectivityResult.mobile
[        ] ==> value = ConnectivityResult.mobile
[+7918 ms] ==> connectionState = ConnectivityResult.mobile
[        ]  ❍ [Monitor] comparing connection state...  
[        ]              connection: true 
[        ]              connectivity result: ConnectivityResult.mobile
[        ] ==> value = ConnectivityResult.mobile
[        ] ==> connectionState = ConnectivityResult.mobile
[        ]  ❍ [Monitor] comparing connection state...  
[        ]              connection: true 
[        ]              connectivity result: ConnectivityResult.mobile
[        ] ==> value = ConnectivityResult.mobile

```

</details>

<details>
<summary>Flutter doctor output</summary>

```
flutter doctor -v
[✓] Flutter (Channel stable, 2.10.5, on macOS 11.6 20G165 darwin-arm, locale ru-RU)
    • Flutter version 2.10.5 at /Users/deev/desktop/development/sdk/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 5464c5bac7 (2 weeks ago), 2022-04-18 09:55:37 -0700
    • Engine revision 57d3bac3dd
    • Dart version 2.16.2
    • DevTools version 2.9.2

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /Users/deev/Library/Android/sdk
    • Platform android-32, build-tools 31.0.0
    • ANDROID_HOME = /Users/deev/Library/Android/sdk
    • ANDROID_SDK_ROOT = /Users/deev/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.11+0-b60-7772763)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2021.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.11+0-b60-7772763)

[✓] VS Code (version 1.66.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.40.0

[✓] Connected device (1 available)
    • Chrome (web) • chrome • web-javascript • Google Chrome 101.0.4951.54

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!

```

</details>

**Comment by dan1229** [comment]
Having the same issue for a while [here](https://github.com/fluttercommunity/plus_plugins/issues/641) - not really sure how this is happening still. Tried digging in a bit but no dice.

Happens on every browser for me.

**Comment by miquelbeltran** [comment]
Should be fixed by #904
